Resources/config/routing.yml

```diff
@@ -66,3 +66,8 @@ pim_customentity_history:
         actionType: history
     requirements:
         id: \d+
+
+pim_customentity_media_rest_post:
+    path: '/rest/customentity_media'
+    defaults: { _controller: pim_custom_entity.media_rest_controller.class:postAction }
```

Comment:
@simoncarre here should be the name of the service `pim_custom_entity.media_rest_controller `, not the class. I got issue here